content/en/docs/reference/command-line-tools-reference/feature-gates/logarithmic-scale-down.md

```diff
@@ -13,6 +13,9 @@ stages:
   - stage: beta
     defaultValue: true
     fromVersion: "1.22"
+  - stage: stable
+    defaultValue: true
+    fromVersion: "1.31"
```

Comment:
Adding `toVersion` to the previous stage would enhance consistency with other feature gate files.

```suggestion
    toVersion: "1.30"
  - stage: stable
    defaultValue: true
    fromVersion: "1.31"
```

Reply:
done